composer.json

```diff
@@ -139,18 +139,18 @@
             "@check-cs",
             "phpunit",
             "@phpstan",
-            "@update-docs"
+            "@docs"
         ],
-        "check-cs": "vendor/bin/ecs check bin packages src tests utils --ansi",
+        "check-cs": "ecs check bin packages src tests utils --ansi",
         "fix-cs": [
-            "vendor/bin/ecs check bin packages src tests utils --fix --ansi",
+            "ecs check bin packages src tests utils --fix --ansi",
```

Comment:
Why this? It's problematic when copy-pasted to modify

Reply:
No problem, @TomasVotruba. I've updated the PR.